xclim/sdba/adjustment.py

```diff
@@ -571,8 +574,8 @@ def _train(
         ref: xr.DataArray,
         hist: xr.DataArray,
         *,
-        ref_params: xr.Dataset = None,
         cluster_thresh: str,
+        ref_params: xr.Dataset = None,
```

Comment:
Not really relevant but just FYI, arguments in a call signature after a `*` _must_ be supplied as kwargs, so the `None` is unnecessary (unless this is an optional argument).

Reply:
Ah, nvm, I see that this is optional. You already know.